_posts/2021-04-21-decoding-from-memory-bug.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: post
+published: true
+title:  "Solidity ABI Decoder Bug For Multi-Dimensional Memory Arrays"
+date:   2021-04-21
+author: Solidity Team
+category: Security Alerts
+---
+
+https://www.certora.com/blog/deserialization.html
+
+On April 5th, 2021, a bug in the Solidity abi decoder v2 was reported by
```

Comment:
We use different spellings for the ABI coder v2. I would suggest to use ``ABI coder v2`` everwhere.
```suggestion
On April 5th, 2021, a bug in the Solidity ABI coder v2 was reported by
```

Reply:
I agree (https://github.com/ethereum/solidity-blog/pull/134#discussion_r616630656). Though I think this one is intentionally mentioning only the decoder because only it is affected.